src/pages/guide/screens/FAQs/FAQs.tsx

```diff
@@ -51,20 +51,31 @@ const FAQs = ({ guideContentRef }: TFAQsProps) => {
                 title={localize('How to register for Deriv P2P?')}
                 variant='underline'
             >
-                <Text as='div' className='mb-[0.5rem]' lineHeight='xl' size='sm'>
-                    <Localize
-                        components={[
-                            <a
-                                className='guide__content-section--link'
-                                href={`${URLConstants.derivAppProduction}/account/proof-of-identity`}
-                                key={0}
-                            />,
-                        ]}
-                        i18n_default_text='Age-verify your account by submitting  <0>proof of identity</0>.'
-                    />
-                </Text>
+                <ul>
+                    <Text as='li' lineHeight='xl' size='sm'>
+                        <Localize
+                            components={[
+                                <a
+                                    className='guide__content-section--link'
+                                    href={`${URLConstants.derivAppProduction}/account/proof-of-identity`}
```

Comment:
Let's add the query param so it redirects back to p2p.
```suggestion
                                    href={`${URLConstants.derivAppProduction}/account/proof-of-identity?platform=p2p-v2`}
```

Reply:
ah sure 👍